persist/sqlite/metrics.go

```diff
@@ -66,7 +68,9 @@ func (s *Store) HostMetrics() (result explorer.HostMetrics, err error) {
 			}
 
 			result.TotalStorage += host.Settings.TotalStorage
+			result.TotalStorage += proto4.SectorSize * host.V2Settings.TotalStorage
```

Comment:
`TotalStorage` and `RemainingStorage` should already expressed in bytes. Shouldn't these totals be moved down inside into the `if !host.V2 {` blocks? It seems like the remaining and total storage of a V2 hosts would count twice towards the total if it has both `Settings` populated.

Reply:
We sort of do this here https://github.com/SiaFoundation/explored/pull/181/commits/542f63bafa2f80850018c0a9e5ae8acae9be1659#diff-f1c64d840d659e37e8885562cee69016a811979ab014f064ea6488baea07d263R362

Reply:
It appears `sia1.euregiohosting.nl` is offline at the moment, causing `TestScan` to fail for me locally. What's the reason we include an actual live host in these tests? We also appear to be sleeping for quite long periods of time, e.g. `time.Sleep(4 * cfg.Timeout)` that's a two minute sleep

Reply:
@chris124567 can you rewrite that test to use ephemeral hosts you spin up yourself rather than live hosts? 
Try to keep it as minimal as possible. e.g. you shouldn't need a full host but only a mocked one which can give you an announcement for itself and listen for RPCSettings and RPCPriceTable requests. Then you can freely pick the settings it should return and check for them rather than relying on a live host.